ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/networking/libp2p/rpc/BlobIdentifier.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright ConsenSys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.spec.datastructures.networking.libp2p.rpc;
+
+import org.apache.tuweni.bytes.Bytes32;
+import tech.pegasys.teku.infrastructure.ssz.containers.Container2;
+import tech.pegasys.teku.infrastructure.ssz.containers.ContainerSchema2;
+import tech.pegasys.teku.infrastructure.ssz.primitive.SszBytes32;
+import tech.pegasys.teku.infrastructure.ssz.primitive.SszUInt64;
+import tech.pegasys.teku.infrastructure.ssz.schema.SszPrimitiveSchemas;
+import tech.pegasys.teku.infrastructure.ssz.tree.TreeNode;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+
+public class BlobIdentifier extends Container2<BlobIdentifier, SszBytes32, SszUInt64> {
+
+  public static class BlobIdentifierSchema
+      extends ContainerSchema2<BlobIdentifier, SszBytes32, SszUInt64> {
+
+    private BlobIdentifierSchema() {
+      super(
+          "BlobIdentifier",
+          namedSchema("blockRoot", SszPrimitiveSchemas.BYTES32_SCHEMA),
```

Comment:
I think this should be `block_root`

Reply:
We already use java style for them in more than 9000 other classes

Reply:
I don't count 9000 cases, very few actually, but, this only affects fields that are sent via rest api.

Reply:
Will chang it to align with the ssz schema definition

Reply:
@tbenr it's "over 9000" meme

Reply:
>  this only affects fields that are sent via rest api
it's internal name, it's only in class, it's not sent anywhere

Reply:
I see your point, RPC\internal objects are following the camelCase but since the majority of them are used in apis they are snake_case. Considering that it only matters when we use them in APIs I tend to give this the priority that format and move them all to snake_case for consistency. But at the end I don't care much, I leave the decision to Stefan :)

Reply:
I changed the RPC internal objects to snake_case in schemas to align more with the spec and be consistent. Think it's cleaner.

Reply:
I was wrong sorry, we have camel case names only in `tech.pegasys.teku.spec.datastructures.networking.libp2p.rpc` and looks like I was there last time and remembered wrong. So snake case is needed here